packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -162,23 +146,16 @@ export function generateGCConfigs(
 	const throwOnInactiveLoad: boolean | undefined = createParams.gcOptions.throwOnInactiveLoad;
 
 	const throwOnTombstoneLoadConfig =
-		mc.config.getBoolean(throwOnTombstoneLoadOverrideKey) ??
-		createParams.gcOptions[gcDisableThrowOnTombstoneLoadOptionName] !== true;
+		mc.config.getBoolean(throwOnTombstoneLoadOverrideKey) ?? true;
 	const throwOnTombstoneLoad =
-		throwOnTombstoneLoadConfig && sweepAllowed && !createParams.isSummarizerClient;
-	const throwOnTombstoneUsage =
-		mc.config.getBoolean(throwOnTombstoneUsageKey) === true &&
-		sweepAllowed &&
-		!createParams.isSummarizerClient;
+		throwOnTombstoneLoadConfig && sweepEnabled && !createParams.isSummarizerClient;
```

Comment:
```suggestion
		!mc.config.getBoolean(throwOnTombstoneLoadOverrideKey) && sweepEnabled && !createParams.isSummarizerClient;
```

Can't we ditch the line above i.e. `throwOnTombstoneLoadConfig` and just do this inline.